doc/source/whatsnew/v1.5.0.rst

```diff
@@ -864,9 +864,13 @@ Bug fixes
 
 Categorical
 ^^^^^^^^^^^
+- Bug in :meth:`.Categorical.view` not accepting integer dtypes (:issue:`25464`)
+- Bug in :meth:`.CategoricalIndex.union` when the index's categories are integer-dtype and the index contains ``NaN`` values incorrectly raising instead of casting to ``float64`` (:issue:`45362`)
```

Comment:
Looks like 2 entries got duplicated here.

Reply:
Strange. I think it's happening when I merge with the main branch.

Reply:
Hi @mroeschke, could you point to the lines where the duplicates are? Thanks

Reply:
Hi @mroeschke, I took your suggestions. Is there anything else in the code that needs work?